CefSharp/ModelBinding/ModelBindingExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static T CreateInstance<T>(this Type type, bool nonPublic = false)
         /// <param name="nonPublic"><see langword="true"/> if a non-public constructor can be used, otherwise <see langword="false"/>.</param>
         public static object CreateInstance(this Type type, bool nonPublic = false)
         {
-            return CreateInstanceInternal(type, nonPublic);
+            return Activator.CreateInstance(type, nonPublic);
```

Comment:
If your going to remove a large block of code, please add a comment to your commit description as to the reason please.